docs/cql/index.rst

```diff
@@ -22,6 +22,7 @@ CQL Reference
    mv
    non-reserved-keywords
    reserved-keywords
+   service-levels
```

Comment:
I think this is the wrong place. We should have a discussion about service level concepts in one place, and cql reference in another (like ddl/dml). But the documentation already mixes them up, so it's okay for now.